packages/zwave-js/src/lib/controller/Controller.ts

```diff
@@ -1573,11 +1636,26 @@ export class ZWaveController
 			);
 		}
 
+		// BUGBUG: fix me
+		// if (options.isLongRange && !this.isLongRange()) {
+		// 	throw new ZWaveError(
+		// 		`Invalid long range inclusion on a non-LR host`,
+		// 		ZWaveErrorCodes.Argument_Invalid,
+		// 	)
+		// }
+		const isLongRange = this.isLongRange();
+
```

Comment:
I couldn't actually get a note to include as long range here.

Morever, we probably want whether we're including as long range or not to be an option/parameter passed in, yeah?